packages/compass-connections/src/connections-manager.ts

```diff
@@ -355,3 +364,18 @@ export class ConnectionsManager extends EventEmitter {
     this.emit(connectionEvent, ...connectionEventParams);
   }
 }
+
+function isAtlasStreamsInstance(
+  adjustedConnectionInfoForConnection: ConnectionInfo
+) {
+  try {
+    return mongodbBuildInfo.isAtlasStream(
+      adjustedConnectionInfoForConnection.connectionOptions.connectionString
+    );
+  } catch (e) {
```

Comment:
```suggestion
  } catch {
```

(I think eslint also complains otherwise about the unused variable?)

Reply:
For some reason it didn't complain, maybe we don't have that rule cause older versions of js didn't let you have a catch without an error  🤷‍♂️, updated along with the copy